src/optionroms.c

```diff
@@ -136,9 +136,19 @@ init_optionrom(struct rom_header *rom, u16 bdf, int isvga)
 
     tpm_option_rom(newrom, rom->size * 512);
 
+    //TODO: Find a way to hide initialisation string of iPXE
+    // It is called everytime and I have no idea how it was hidden before.
+    // For now it can be commented out and seems to work good.
```

Comment:
@macpijan is it something that you implemented ? Have you got any ideas ?

Reply:
@pietrushnic did You mean me or @macpijan ?

Reply:
@miczyg1 I mean @macpijan - I'm pretty sure he did initial implementation. Git blame ?

Reply:
No, It just also sounded like question directed to me. Wanted to ensure myself only.

Reply:
@pietrushnic Yes I can remember doing pxe enable/disable. @miczyg1 Which is initialization string we want to hide?

Reply:
@macpijan the string is something like that:
```
iPXE (http://ipxe.org) 00:00.0 C000 PCI2.10 PnP PM
Press Ctrl-B to configure iPXE (PCI 00:00.0)...
```
Whenever callrom() function is called, the string is printed out on the screen/console.